### PR TITLE
[Documentation] Document process

### PR DESCRIPTION
**Pull request by VWoeltjen**
Addresses #223, documents sprint process/structure.

### Author Checklist

1. Changes address original issue? Y
2. Unit tests included and/or updated with changes? N/A &ast;
3. Command line build passes? Y
4. Expect to pass code review? Y

&ast; Changes are to documentation only, out of scope for unit testing

### CODE REVIEW

File: docs/src/process/index.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+higher-level strategic decisions about desired functionality
+and characteristics of the software, major external milestones,
+and so forth.
+
+In the absence of a dedicated project manager, this role may be rotated
+among members of the development team on a per-sprint basis.
+
+Responsibilities of the project manager including:
+
+* Maintaining (with agreement of stakeholders) a "road map" of work
```
Comment (akhenry) [inline docs/src/process/index.md]: We should publish our roadmap for OpenMCTWeb on github (or in the wiki).

File: docs/src/process/index.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+  usually expressed as "themes",
+  with just enough specificity to gauge feasibility of plans,
+  relate work back to milestones, and identify longer-term
+  dependencies.
+* Determining (with assistance from the rest of the team) which
+  issues to work on in a given sprint and how they shall be
+  assigned.
+* Pre-planning subsequent sprints to ensure that all members of the
+  team always have a clear direction.
+* Scheduling and/or ensuring adherance to
```
Comment (akhenry) [inline docs/src/process/index.md]: ~~adherance~~ adherence

File: docs/src/process/index.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  [process points](#process-points).
+* Responding to changes within the sprint (shifting priorities,
+  new issues) and re-allocating work for the sprint as needed.
+
+## Sprint Calendar
+
+Certain process points are regularly scheduled in the sprint cycle.
```
Comment (akhenry) [inline docs/src/process/index.md]: Link to process points

**Comment by akhenry** [comment]
This is awesome, great idea putting this up on Github. 

One very minor spelling correction, and a suggested link. Also, are we able to publish the roadmap?

**Comment by VWoeltjen** [comment]
Thanks for catching that - pushed changes to correct spelling error and add useful link. Filed #296 for road map; think it would be useful to release this but there are some open questions (do we include info for closed-source projects? if not, do we want to maintain this across separate documents? does it make sense to include time-bound documentation at all?) so want to handle that separately.

**Comment by akhenry** [comment]
Yeah, the timeliness of the roadmap makes it tricky. Perhaps the best approach would be to include it in the wiki? That way we can easily update it including the progress that we have made against the roadmap.

**Comment by akhenry** [comment]
Changes look good, merging.

### Reviewer Checklist
* Changes appear to address issue? __N/A__
* Appropriate unit tests included? __N/A__
* Code style and in-line documentation are appropriate? __N/A__
* Commit messages meet standards? __Y__

**Pull request merged**